jina/serve/networking.py

```diff
@@ -166,8 +166,10 @@ def __init__(self, address, channel, summary):
 
         # This has to be done lazily, because the target endpoint may not be available
         # when a connection is added
-        def _init_stubs(self):
-            available_services = GrpcConnectionPool.get_available_services(self.address)
+        async def _init_stubs(self):
+            available_services = await GrpcConnectionPool.get_available_services(
+                self.channel
```

Comment:
this is the important change as we use the existing `self.channel` and not create a new one from `address`